.github/workflows/maven.yml

```diff
@@ -12,20 +12,18 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
-    branches: [ "main" ]
+    branches: [ "*" ]
```

Comment:
Is there any reason you don't want it set to `main`?

Reply:
It's been set to '*' rather than main because I had branched off Stephen's branch originally before anything had been pushed to main, so the build actions weren't running properly whenever I was making my pull requests. I had left it as that just in case this ever happened again in the future.